packages/teams-js/src/private/externalAppCardActions.ts

```diff
@@ -113,14 +113,20 @@ export namespace externalAppCardActions {
   /**
    * @beta
    * @hidden
-   * Delegates an Adaptive Card Action.OpenUrl request to the host for the application with the provided app ID
+   * Delegates an Adaptive Card Action.OpenUrl request to the host for the application with the provided app ID.
+   * Currently `processActionOpenUrl` is supported only for composeExtensions and not for plugins
```

Comment:
That isn't quite true. Generic URLs are supported for plugins.

Reply:
But can Teams process them without knowing the 'fromElement'? From my understanding, Teams cannot process any urls for plugins without knowing the url is for plugin. And they are not going to know plugin vs composeExtension until M365 chat send the info